### ISSUE TITLE
Master

### ISSUE THREAD

**Issue by Ken125pig** [w3c/wcag-act#523]
The ACT-Rules community group has completed work on {{rule name}}. We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: ...
**Rule id**: ...
**Link to the rule**: ...
**License**: ... 
**Implementors**:
- {{Implementor name}}
- {{Implementor name}}
- ...

If you'd like to contact me, please e-mail me at myemail@mycompany.com. (optional: You can also reach me on the W3C Community Slack, my name there is <<name>>)
